libglusterfs/src/glusterfs/glusterfs.h

```diff
@@ -315,6 +315,7 @@ enum gf_internal_fop_indicator {
 #define DHT_SKIP_OPEN_FD_UNLINK "dont-unlink-for-open-fd"
 #define DHT_IATT_IN_XDATA_KEY "dht-get-iatt-in-xattr"
 #define DHT_MODE_IN_XDATA_KEY "dht-get-mode-in-xattr"
+#define DHT_RENAME_FOP_KEY "dht-rename-fop"
```

Comment:
I would use a more generic name unrelated to DHT, like `GF_MKNOD_REPLACE_KEY` for example.

Reply:
done